feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/view/StakeSummaryView.kt

```diff
@@ -52,11 +50,7 @@ class StakeSummaryView @JvmOverloads constructor(
         }
 
         if (status is Status.Waiting) {
-            if (timer != null) timer?.cancel()
-
-            timer = stakeSummaryStatusHelper.startTimer(status.timeLeft)
-
-            timer?.start()
+            stakeSummaryStatusHelper.startTimer(status.timeLeft)
         } else {
             stakeSummaryStatusHelper.text = status.extraMessage
```

Comment:
I think we should stop the timer here (suggest writing a separate method `stopTimer()`, which will extract timer from tag and stop it). Otherwise, if a new era comes before the timer is finished, timer may overwrite text once more

Reply:
Done